src/main/java/io/strimzi/test/container/StrimziKafkaContainer.java

```diff
@@ -30,122 +32,62 @@
  * The additional configuration for Kafka broker can be injected via constructor. This container is a good fit for
  * integration testing but for more hardcore testing we suggest using @StrimziKafkaCluster.
  */
+@SuppressWarnings("deprecation")
```

Comment:
You should have a comment about why is it here - to remember what was deprecated to remove this.